diff --git a/docker/Dockerfile.base b/docker/Dockerfile.base
index 1b912e0..7feb87c 100644
--- a/docker/Dockerfile.base
+++ b/docker/Dockerfile.base
@@ -28,6 +28,7 @@ RUN wget https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.
     dpkg -i *.deb && \
     rm -f *.deb && \
     apt-get update && \
+    pip install --upgrade pip==9.0.3 && \
     apt-get install -y wget build-essential make gcc binutils python-dev libffi-dev libssl-dev git && \
     pip install -r /tmp/requirements.txt && \
     apt-get purge -y wget build-essential make gcc binutils python-dev libffi-dev libssl-dev git && \
diff --git a/envoy/go/envoyd/envoyd.go b/envoy/go/envoyd/envoyd.go
index 6942df3..51098ae 100644
--- a/envoy/go/envoyd/envoyd.go
+++ b/envoy/go/envoyd/envoyd.go
@@ -32,7 +32,7 @@ import (
 	"flag"
 	"bufio"
 	consulapi "github.com/hashicorp/consul/api"
-	etcdapi "github.com/coreos/etcd/clientv3"
+	etcdapi "go.etcd.io/etcd/clientv3"
 )
 
 // DATA STRUCTURES
